content/posts/newsletter/2021-07-05.md

```diff
@@ -0,0 +1,184 @@
+
+## Events
+
+[XMPP Office Hours each week](https://wiki.xmpp.org/web/XMPP_Office_Hours) - Also, checkout our new [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)!
+
+[Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): Monthly Meeting of XMPP Enthusiasts in Berlin - always 2nd Wednesday of the month.
+
+## Videos
+
+[Demo: ad-hoc commands and data forms in Mellium](https://www.youtube.com/watch?v=C2oyAfJeqno) - Sam Whited demos the new support for XEP-0050: Ad-Hoc Commands and XEP-0004: Data Forms in the Mellium XMPP library.
+
+## Articles
+
+Martin Dosch wrote a short blog post about [how to use the "Jabber-ID" email header with neomutt](https://blog.mdosch.de/2021/06/13/jabber-email-header/) [DE]. The setting of this header is also described in the [xmpp.org wiki](https://wiki.xmpp.org/web/Jabber_Email_Header).
+
+Axel Reimer published a short installation and configuration guide for Monal 5.0. It is available in [English](https://eversten.net/en/blog/monal/) and [German](https://eversten.net/blog/monal/).
+
+Vaxbot, the XMPP based vaccine bot [has been featured on Canadian National News](https://monal.im/blog/vaxbot-on-canadian-national-news/).
+
+Software freedom conservancy [moved their chat from IRC to XMPP](https://sfconservancy.org/blog/2021/jun/21/xmpp-chat/) using [Snikket](https://snikket.org/).
+
+Ingrid’s Space has thoughts on [why decentralised applications don’t work](https://ingrids.space/posts/why-distributed-systems-dont-work/).
+
+## Software news
+
+### Clients and applications
+
+UWPX [v.0.33.0.0](https://github.com/UWPX/UWPX-Client/releases/tag/v.0.33.0.0) has been released. It includes support for [XMPP Providers](https://invent.kde.org/melvo/xmpp-providers/-/tree/master) for convenient auto-complete and registration of new accounts to XMPP servers. Besides that, this release fixes a few bugs related to stream close and open message parsing.
+![UWPX](https://user-images.githubusercontent.com/11741404/124069653-cf8dc600-da3c-11eb-86b7-324cbc6ac26f.png)
+
+[Gajim Development News](https://gajim.org/post/2021-06-27-development-news-june/): This month brought some fixes for themes and spell checking. 
+Development on the new Gajim version made a big step forward: file previews are working again and more important they're now a core integrated feature, they look even better while also being privacy respecting for those anonymous chats.
+
+Monal 5.0 [has been released](https://monal.im/blog/monal-with-group-chat-and-voice-messages/), featuring support for group chats (not yet OMEMO encrypted, will come next) and sending voice messages and videos, but also a lot of internal code refactoring and bug fixes. 
+![Monal](https://monal.im/wp-content/uploads/2021/06/Monal1.png)
+
+Blabber.IM XMPP service has been discontinued and has shut down its server. On June 25, the official blabber.im website reported that they were discontinuing support for their XMPP server.
```

Comment:
June 25th [...] [blabber.im](https://blabber.im) [...] their XMPP server due to personal reasons.